src/databricks/labs/ucx/cli.py

```diff
@@ -687,16 +687,7 @@ def migrate_tables(
     for workspace_context in workspace_contexts:
         deployed_workflows = workspace_context.deployed_workflows
 
-        try:
```

Comment:
@FastLee : You added this in #2929. From the context of the PR and linked issue, I cannot determine why it is added in that PR. Do you remember why?